nevergrad/optimization/experimentalvariants.py

```diff
@@ -302,6 +303,9 @@
 ProgONOPO13 = ConfSplitOptimizer(num_optims=13, progressive=True, multivariate_optimizer=OptimisticNoisyOnePlusOne).set_name(
     "ProgONOPO13", register=True
 )
+ProgONOPOInf = ConfSplitOptimizer(num_optims=100000, progressive=True, multivariate_optimizer=OptimisticNoisyOnePlusOne).set_name(
```

Comment:
You should define this directly in `splitters`, no need to register it
for instance:
```python
def splitters() -> tp.List[Optim]:
    optims: tp.List[Optim] = [x for x in ng.optimizers.registry if "Prog" in x]
    optims.append(ConfSplitOptimizer(num_optims=100000, progressive=True, multivariate_optimizer=OptimisticNoisyOnePlusOne).set_name("ProgONOPOInf"))
    return optims
```

Also, no need to sort the list, this is done by `get_optimizers`